azure-stack-go-sdk/azure-stack-go-sdk-profiles.md

```diff
@@ -0,0 +1,302 @@
+---
+title: Installing and use Azure Go SDK profiles on Azure Stack
+description: How to install and use Azure Go SDK profiles on Azure Stack environment.
+keywords: azure, azure stack, sdk, go, golang, azure sdk
+author: seyadava
+ms.author: seyadava
+ms.date: 03/22/2018
+ms.topic: article
+ms.devlang: go
+manager: knithinc
+---
+
+# What is a profile?
```

Comment:
The first H2 of the article should be an overall summary, and must match the title of the article. Please add one.

Reply:
Include a newline after every H2.